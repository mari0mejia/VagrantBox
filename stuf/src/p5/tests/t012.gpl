
int    lhs_int;
double lhs_double;

int    rhs_int=22;
double rhs_double=33.33;

on hkey
{
  print("Minus-assign. h-key pressed");
  lhs_int    -= rhs_int;
  lhs_double -= rhs_double;
}

on jkey
{
  print("Minus-assign. j-key pressed");
  lhs_double -= rhs_int;
}
