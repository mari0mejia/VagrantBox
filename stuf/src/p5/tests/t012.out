gpl.cpp::main()
  input file(tests/t012.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
double lhs_double = 0
int lhs_int = 0
double rhs_double = 33.33
int rhs_int = 22
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().
print[10]: Minus-assign. h-key pressed
print[17]: Minus-assign. j-key pressed

User quit program. Printing the symbol table.
double lhs_double = -55.33
int lhs_int = -22
double rhs_double = 33.33
int rhs_int = 22
